nilearn/_utils/data_gen.py

```diff
@@ -967,15 +963,27 @@ def create_fake_bids_dataset(
     return bids_dataset_dir
 
 
-def _check_tasks(tasks):
-    for task_ in tasks:
-        bids.validate_label(task_)
-
-
 def _check_entities(entities: dict) -> None:
```

Comment:
```suggestion
def _verify_labels_are_alnum(entities: dict) -> None:
```

explains the validation;

Reply:
fixed